docs/sources/write/grot-guides/index.md

```diff
@@ -0,0 +1,134 @@
+| --------- | --------------------------------------------------------------------------- | -------- |
+| `name`    | The name of the guide Markdown file in the website `content/guides` folder. | yes      |
+| `title`   | A custom guide title to override the default guide title.                   | no       |
+| `text`    | Custom body text to override the default guide text.                        | no       |
+| `button`  | Custom button copy to override the default guide button copy.               | no       |
+
+If you don't set custom title, text, and button values, the guide introduction screen uses the following default text:
+
+![Grot guide introduction screen with default copy](https://grafana.com/media/docs/writers-toolkit/grot-guide.png)
+
+## Structure of a Grot guide
+
+Guide content consists of various YAML structures in Markdown front matter for the guide meta, header, welcome, and screens.
+
+### Meta
+
+The meta sections covers general information about the guide. Customize the `name`, `api`, and `title` fields, and leave the rest as is:
+
+```yaml
+# name that matches the filename and name used to embed
+name: <GUIDE NAME>
+# match the name field
+api: <GUIDE NAME>
+title: <GUIDE TITLE>
+type: guides
+layout: single
+# Hugo rendering options
+_build:
+  render: false
+  list: true
+```
+
+## Header
+
+The header structure defines the guide's image and image spacing. The following example sets a `src`, `alt`, and image dimensions:
+
+```yaml
+header:
+  image:
+    src: /media/guides/grafana-guides-whichgrafana-header.svg
+    alt: Grot metrics wizard
+    width: 221
+    height: 131
+```
+
+## Welcome
+
+A guide needs a single welcome object, defined with a `welcome` field and the following attributes. Define a list of `ctas` with at least one item which points to the next `screen_id`.
+
+```yaml
+welcome:
+  type: welcome
+  title: Let's get started with application instrumentation
+  body: Answer a few questions and Grot can help you find the documentation you need to instrument your application.
+  ctas:
+    - text: Let's go!
+      screen_id: instrument
+```
+
+## Screens
+
+Define subsequent screens as a list of objects under a `screens` field:
+
+```yaml
+screens:
+  - type: question
+    id: ...
+
+  - type: result
+    id: ...
+```
+
+### Question
+
+A question screen is a branch node in a decision tree and presents one or many options to further screens. A question screen has the following structure:
+
+```yaml
+screens:
+  ...
+
+  - type: question
+    id: instrument
+    title: What programming language do you want to instrument?
+    options:
+      - text: JVM (Java, Scala, Kotlin)
+        screen_id: jvm
+      - text: .Net
+        screen_id: dotnet
+      - text: Node.js
+        screen_id: nodejs
+```
+
+### Result
+
+A result screen is a leaf node in a decision tree and terminates with one or many links. A question screen has the following structure:
+
+```yaml
+screens:
+  ...
```

Comment:
https://grafana.com/docs/writers-toolkit/write/style-guide/write-for-developers/#wherever-possible-ensure-the-partial-example-is-copy-pasteable:~:text=Don%E2%80%99t%20use%20an%20ellipsis%20(%E2%80%A6)%20or%20three%20periods%20(...)%20to%20omit%20information.

```suggestion
```